src/scripting/text.cpp

```diff
@@ -16,10 +16,19 @@
 
 #include "scripting/text.hpp"
 
+#include <stdexcept>
```

Comment:
What do you need that include for?

Reply:
It seems to be leftover from another thing I was working on.  I'll fix it.